README.md

```diff
@@ -1,6 +1,10 @@
 blpapi-http
 ===========
 
+NOTE: The version of this project that is running for hackathons can be found
+in the [Hackathon branch]; go there for docs and examples that match the
```

Comment:
[hackathon](http://github.com/bloomberg/blpapi-http/tree/hackathon) - I would just inline it here like this and remove it from the listings below.